web/src/main/java/io/fabric8/launcher/web/endpoints/BoosterCatalogEndpoint.java

```diff
@@ -136,4 +136,4 @@ public Response reindex(@QueryParam("token") String token) {
         boosterCatalogFactory.reset();
         return Response.ok().build();
     }
-}
\ No newline at end of file
+}
```

Comment:
This doesn't look like it belongs to this PR, can you remove from this PR, please?